sql/core/pom.xml

```diff
@@ -39,7 +39,7 @@
     <dependency>
       <groupId>com.univocity</groupId>
       <artifactId>univocity-parsers</artifactId>
-      <version>2.1.0</version>
+      <version>2.1.1</version>
```

Comment:
this will fail the dependency check - you will need to update the dep list. Jenkins will tell you what to do.